### ISSUE TITLE
Which Github is current?

### ISSUE THREAD

**Issue by BuildAndDie** [openlegend/core-rules#463]
I'm so confused. Things that are in the most recent alpha PDF are not showing up the same as the print tree edition, and neither are matching the regular github.

When looking up stuff I'm having to check all 3 sources. Is there another tree we don't know about on github?

**Comment by brianfeister** [comment]
Nope shouldn't be any other. Provide some examples so I can understand the problem. This should be current: 

github.com/openlegend/core-rules/tree/print-edition

**Comment by BuildAndDie** [comment]
ok, I think I see the issue, see, I'm going into the:

https://github.com/openlegend/core-rules/tree/print-edition/feats

to read the feats.yml, and there it lists Destructive Trance with the prereq as only Might

But if you go into the core folder > src folder > 04-feats folder

then it lists it the same as the alpha.

**Comment by jcte02** [comment]
`.yml` files in the `print-edition` are outdated, since they serve no purpose (`.yml` files are for the site to parse).
I had not the time to look at the new alpha, but right now the `master` branch reflects the same changed as the `print-edition` branch in the places you are used to (i.e. the `.yml` files).
The main site is still outdate tho, it is still referencing changes until May 10th. But this is a triefling matter, as since the `master` branch is up to date, all that is needed is a bit of reference update.

**Comment by BuildAndDie** [comment]
So, speaking on this topic:

Awesome that everything got sent to the printer!!! Do we have any ETA on when the "regular" github will get updated to match the print-edition tree? And from there for the website.

Asking b/c handling questions in discord that deal with the website being wrong, and lately, people even referencing the github, which is wrong, etc.

**Comment by jcte02** [comment]
@BuildAndDie the `master` branch is on par with the `print-edition` branch. The only thing left to do is for PR #464 to be merged, which contains the proofreading edits. So "regular" github is already updated.

**Comment by jcte02** [comment]
@BuildAndDie So I did a quick check and it turns out that the "regular" github (the `master` branch) was already on par when you opened this issue. Print edition edits were merged into master with PR #460, if you are interested in looking into that.

TL;DR: "regular" github is right, site is not

**Comment by BuildAndDie** [comment]
I wish that was the case, but it isn't.
Biggest example
The regular Github lists:
03-feats
06-banes-boons

Print Edition lists:
03-banes-boons
04-feats

Under the feats.yml in the regular github:

Destructive Trance:
missing the "any:" to indicate that you need any of the following, instead will say you need all of the following.

maybe those are the only things, I'm pretty sure there were a few others, but I can't remember them off the top of my head right now. I'll try and think of them and mention it here I suppose.

I looked over the PR 464 you mentioned and didn't see those there.

**Comment by jcte02** [comment]
Whoops! The `Destructive Trance` one is in fact a typo. Thanks for spotting it out. It seems that while updating the `master` branch, I just copied the new attributes in forgotting to add the `any` specificator. The fix is now included in #464. 

As I wrote in my last reply, the bulk update to the regular github was done via #460. As I manually applied the patches from the print version, there may be errors around. I am still human, after all. Nevertheless, I am still confident that the almost all the changes from the `print-edition` were merged into `master`. If you don't trust my word and prefer to investigate more deeper, I redirect you #460, where you can see which of the `print-edition` changes were added to `master`.

I didn't pay much attention to the modified chapter order, as I tought it was only for the print version? I may be wrong on this one.

**Comment by jcte02** [comment]
If you spot other errors, please write them there, and I will fix them via #464.

**Comment by BuildAndDie** [comment]
ah, ok, that's mainly why I was asking, didn't know if I should make new issues or not, since I knew stuff was waiting to be merged to match the stuff sent to print.

The chapter order was something to help make the flow of character creation go smoother, since someone would be reading through the chapters in order. It was suggested awhile back, but I believe it was intended for both print and the website (the difference being the website has a short blurb in teh feats section and then the link (.yml file) to view all the feats).

**Comment by jcte02** [comment]
Got it. Chapters are now ordered correctly in #464.

**Comment by brianfeister** [comment]
Thanks guys. I just moved the site from Openshift / Redhat over to GCE now. Hoping I didn't bork the DNS setup or some such thing as I'll be at the Ren Faire tomorrow. 

If DNS does break, at least two people know what happened ;)

**Comment by ucffool** [comment]
Welp, it's ~broken~... er, propagating. Worth checking in before you leave. :)

**Comment by brianfeister** [comment]
fuck

**Comment by brianfeister** [comment]
What a headache 😢  

There used to be a DNS level redirect from `http://openlegendrpg.com` -> `http://www.openlegendrpg.com` GCE doesn't want it set up that way so I deleted that DNS record but hopefully it's just lagging on the TTL because I did delete it after I set up the other DNS... or did I?

**Comment by brianfeister** [comment]
So **pissed** right now. 😡 

Fellow devs feel free to let me know if you see anything amiss:

![image](https://user-images.githubusercontent.com/806536/31042953-87bd834a-a581-11e7-9722-21f6b31cb2c5.png)

**Comment by brianfeister** [comment]
... confusing. GCE was showing me a running version of the app, but didn't really make it explicit that it was just "sandbox" rather than production. Running `gcloud app deploy ./app.yaml` now with fingers crossed it comes online in the next little bit here.

**Comment by ucffool** [comment]
Upgraded to a server error, that's a plus.
> Error: Server Error
> The server encountered a temporary error and could not complete your request.
> Please try again in 30 seconds.
